app/render/settings.ts

```diff
@@ -273,7 +275,21 @@ async function nightPDFSettings() {
 		);
 
 		// @todo: add event listener to save button, check for duplicates
-		// ...
+		saveButton?.addEventListener(
+			"click",
+			() => {
+				const values = document.getElementById(
```

Comment:
I know this is WIP, but you can use `newKeybind`. I'll see if I have some time tonight to refactor `type Keybind` into an class